linux/sound.cpp

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************\
- * Copyright (c) 2004-2020
+ * Copyright (c) 2004-2021
```

Comment:
There’s something wrong here!

Reply:
>  or would you really use the "last edited" date? I don’t see too much benefits from not just using the current year.

This was the intention and is what's currently implemented, yes.
Having read some more, the benefit of doing this 100% accurate seems low and the recommendation is to update all files consistenly.

I guess I should drop this logic again to simplify things.

https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html

Reply:
The updated logic no longer inserts file-specific dates, it always updates to the current year. Resolving.